server/swap/state.go

```diff
@@ -33,8 +33,6 @@ type State struct {
 	MatchTrackers map[order.MatchID]*matchTrackerData
 	// Swapper.orders.orderMatches
 	OrderMatches map[order.OrderID]*orderSwapStat
-	// Swapper.liveAckers
-	LiveAckers map[uint64]*msgAckers
 	// Swapper.liveWaiters
 	LiveWaiters map[waiterKey]*handlerArgs
```

Comment:
We can probably get rid of `LiveWaiters` eventually too, but leaving for now.

Reply:
If the client resubmits their init or redeem on reconnect?

Reply:
If we have a live coin waiter, we haven't acked the client's request, so it'll time out and retry again anyways, I think.

Reply:
Makes sense, client will reconnect, check match_status, and if needed do their 'init' or 'redeem' again.  That the plan?